scripts/release-go-tags.sh

```
@@ -0,0 +1,311 @@
+# discussion and RELEASE.md.
+required_modules=(
+    "adapters/common"
+    "bamlutils"
+    "dynclient"
+    "dynclient/baml-patched"
+    "introspected"
+    "worker"
+    "workerplugin"
+)
+
+first_party_prefix="github.com/invakid404/baml-rest"
```

Shellcheck (0.11.0)

[warning] first_party_prefix appears unused. Verify use (or export if used externally).

(SC2034)